create_project.py

```diff
@@ -757,50 +757,97 @@ def parse_donor_organism_data_from_xlsx(wb):
             print(f'{project_key} section not found in the data!')
     return data
 
+def write_json(file_name, json_content):
+    with open(file_name, 'w') as f:
+        f.write(json.dumps(json_content, indent=4))
+    print(f'{file_name} successfully written.')
 
-def write_project_json(project_json, output_dir):
-    with open(f'{output_dir}/project_0.json', 'w') as f:
-        f.write(json.dumps(project_json, indent=4))
-    print(f'"{output_dir}/project_0.json" successfully written.')
+
+def load_json(file_name):
+    with open(file_name, 'r') as f:
+        json_content = json.loads(f.read())
+    return json_content
+
+
+def create_json_if_not_present(file_name):
+    if not os.path.exists(file_name):
+        with open(file_name, 'w') as f:
+            f.write('{}\n')
+
+
+def update_file_uuid_config_file(file_name, file_uuid, output_dir):
```

Comment:
Wouldn't this be simpler if `.upload()` accepted a `dict` with the mapping instead of a file containing the mapping?

Reply:
I'll change it to allow both.